cypress/support/index.d.ts

```diff
@@ -0,0 +1,97 @@
+/// <reference types="cypress" />
+
+declare namespace Cypress {
+    interface Chainable<Subject> {
```

Comment:
Inspirert av [Cypress sin example-todomvc](https://docs.cypress.io/api/cypress-api/custom-commands#5-Write-TypeScript-definitions) for å få autocomplete ute i testene.